core-services/various_rbac/admin_ci-operator_roles.yaml

```diff
@@ -761,7 +761,7 @@ items:
   apiVersion: rbac.authorization.k8s.io/v1
   metadata:
     name: image-tagger
-    namespace: openshift-acme
+    namespace: openshift-azure
```

Comment:
Should this be `images-openshift-acme` as well?

Reply:
no idea how I managed to do that, thx